ground/gcs/src/plugins/opmap/opmapgadgetwidget.cpp

```diff
@@ -265,6 +265,18 @@ OPMapGadgetWidget::OPMapGadgetWidget(QWidget *parent) : QWidget(parent)
         }
     }
 
+    // Update the home location if it is already set:
+    if (obum) {
+        bool set;
+        double LLA[3];
+        if (obum->getHomeLocation(set, LLA) == 0) {
+            if (LLA[0] != 0 && LLA[1] != 0 && LLA[2] != 0)
```

Comment:
We don't care about it in this particular context (the set flag is only used to check that we have set the home location in the UAV, unless I misunderstand its use ? )